tox.ini

```diff
@@ -369,7 +369,7 @@ commands_pre =
   python scripts/eachdist.py install --editable --with-test-deps
 
 commands =
-  python scripts/eachdist.py lint --check-only
+  python scripts/eachdist.py lint
```

Comment:
will this command still fail on fixing styling commands? I think the reason this was --check-only was to use that tox command for CI, rather than for users trying to run lint / formatting rules.

Reply:
Sorry left it in by accident =p